src/agent/agent.c

```diff
@@ -30,6 +31,69 @@ static bool
                                 void *userdata,
                                 free_func_t free_userdata);
 
+#define AGENT_HEARTBEAT_INTERVAL_USEC (1000000)
+
+static int agent_reset_heartbeat_timer(Agent *agent, sd_event_source **event_source);
+
+static int agent_heartbeat_timer_callback(sd_event_source *event_source, UNUSED uint64_t usec, void *userdata) {
+        Agent *agent = userdata;
+        int r = 0;
+
+        assert(event_source);
+
+        // static uint32_t count;
+        // printf("%s(): heartbeat count = %d\n", __func__, count++);
+
+        r = sd_bus_emit_signal(
+                        agent->peer_dbus,
+                        INTERNAL_AGENT_OBJECT_PATH,
+                        INTERNAL_AGENT_INTERFACE,
+                        "Heartbeat",
```

Comment:
Consider moving the value to a `#define`

Reply:
See https://github.com/containers/hirte/issues/109

Reply:
Yep!  I'll add the following to protocol.h:
```
/* Agent to Hirte heartbeat signals */

#define AGENT_HEARTBEAT_INTERVAL_USEC (0020000)

#define AGENT_HEARTBEAT_SIGNAL_NAME "Heartbeat"
```